libqtile/window.py

```diff
@@ -602,6 +629,22 @@ def _items(self, name):
     def _select(self, name, sel):
         return None
 
+    def to_relative_top(self, **kwargs):
+        if self.group:
+            self.group.windows.remove(self)
+            self.group.windows.append(self)
+            self.group.assure_correct_layer(self, **kwargs)
+        else:
+            self.window.configure(stackmode=StackMode.Above, **kwargs)
+
+    def to_relative_bottom(self, **kwargs):
```

Comment:
Let's drop it if we don't use it?

Reply:
I though it was fitting to have it. It's gone.

Reply:
I'll leave the conversation open in case you actually wanted me to remove the `else`, too. That wouldn't be a good idea, though. Windows without groups exist (I might have seen that in some test).

Reply:
If you are going through these changes commit by commit, the `to_relative_top`-method will be used quite a bit in later commits. I considered it "setup" and thus included it here. `to_relative_bottom`, however, isn't used anywhere.

Reply:
Then it should be implemented in the commit that actually uses it :)

Reply:
I had that coming.^^ I'll move it when I merge in all the small fixes that resulted from your review.